lib/routes/ieee/author.ts

```diff
@@ -0,0 +1,101 @@
+import { Route } from '@/types';
+import { getCurrentPath } from '@/utils/helpers';
+const __dirname = getCurrentPath(import.meta.url);
+
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import path from 'node:path';
+import { art } from '@/utils/render';
+
+export const route: Route = {
+    path: '/author/:aid/:sortType/:count?',
```

Comment:
This should be `path: '/author/:aid/:sortType?/:count?',` since sortType is not mandantory.

Reply:
thanks for your reviewed.